debian/control

```diff
@@ -0,0 +1,28 @@
+Source: libplayback
+Section: libs
+Priority: optional
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
+Build-Depends: debhelper (>= 5), libtool, pkg-config, libdbus-1-dev
+
+Package: libplayback-1-0
+Architecture: any
+Section: libs
+Priority: optional
+Depends: ${shlibs:Depends}
+Description: OSSO Playback client library.
+
+Package: libplayback-1-dev
+Architecture: any
+Section: devel
+Priority: optional
```

Comment:
Should Section be removed for -dev and -dbg packages as well? Source part has "Section: libs". I checked Maemo binary package, it has "Section: libdevel" for -dev.